src/main/java/dev/mccue/resolve/maven/MavenCentralDownloader.java

```diff
@@ -0,0 +1,92 @@
+package dev.mccue.resolve.maven;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.nio.channels.Channels;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * Creates an object from a specified path at maven central. Allows for the downloading of POMS and Jars.
+ */
+public class MavenCentralDownloader {
+
+    private final String mavenBaseURL = "https://repo.maven.apache.org/maven2/";
+    private final String pomExtension = ".pom";
+    private final String jarExtension = ".jar";
```

Comment:
constants generally are `private static final` or `public static final` and are `NAMED_LIKE_THIS`

so

```java
    private final String MAVEN_BASE_URL = "https://repo.maven.apache.org/maven2/";
```

Reply:
fixed